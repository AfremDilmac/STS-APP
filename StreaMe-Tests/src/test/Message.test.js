import Message from "../components/Message";
import { shallow } from "enzyme";
import { AuthContext } from "../context/AuthContext";
import { ChatContext } from "../context/ChatContext";

test("should populate right message", () => {
  const message = {
    senderId: "wer-23-we-ewwer",
    text: "any message from user",
    img: "img-url",
  };

  const wrapper = shallow(
    <AuthContext.Provider>
      <ChatContext.Provider>
        <Message message={message} />
      </ChatContext.Provider>
    </AuthContext.Provider>
  );

  expect(wrapper).toMatchSnapshot();
});
