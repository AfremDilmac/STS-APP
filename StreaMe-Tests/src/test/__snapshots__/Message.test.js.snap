// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should populate right message 1`] = `
<ContextProvider>
  <Message
    message={
      Object {
        "img": "img-url",
        "senderId": "wer-23-we-ewwer",
        "text": "any message from user",
      }
    }
  />
</ContextProvider>
`;
